build.gradle

```diff
@@ -30,6 +30,7 @@ ext.stdlibCryptoVersion = project.stdlibCryptoVersion
 ext.stdlibLogVersion = project.stdlibLogVersion
 ext.stdlibRegexVersion = project.stdlibRegexVersion
 ext.stdlibTimeVersion = project.stdlibTimeVersion
+ext.stdlibUrlVersion = project.stdlibUrlVersion
 // Transitive dependencies
 ext.stdlibTaskVersion = project.stdlibTaskVersion
 ext.stdlibConstraintVersion = project.stdlibConstraintVersion
```

Comment:
We do not need these `ext` variable definitions in the `build.gradle` file

Reply:
Removed with [59fd3b5](https://github.com/ballerina-platform/module-ballerina-oauth2/pull/915/commits/59fd3b522b618d75e07fe3a8fa95b36c724d9dbe)